controller/appcontroller.go

```diff
@@ -36,9 +36,6 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/util/workqueue"
 
-	// make sure to register workqueue prometheus metrics
-	_ "k8s.io/component-base/metrics/prometheus/workqueue"
```

Comment:
About why this line doesn't work, because it registers to another metrics registry https://github.com/kubernetes/component-base/blob/master/metrics/prometheus/workqueue/metrics.go#L99 so that we cannot query those metrics when serving `/metrics`

Reply:
Adding commit-specific link in case the file changes in master: https://github.com/kubernetes/component-base/blob/f57281d7c18ff3a9d1c2404bf97f4ca70bdcb655/metrics/prometheus/workqueue/metrics.go#L99